src/client/pages/Download/index.tsx

```diff
@@ -1,55 +1,26 @@
-import styled from '@emotion/styled'
-import { colorsV3 } from '@hedviginsurance/brand/dist'
-import { SemanticEvents } from 'quepasa'
 import React from 'react'
+import { SemanticEvents } from 'quepasa'
 import Helmet from 'react-helmet-async'
 import { Mount } from 'react-lifecycle-components/dist'
-import { LoadingPage } from 'components/LoadingPage'
 import { TopBar } from 'components/TopBar'
 import { Page } from 'components/utils/Page'
 import { SessionTokenGuard } from 'containers/SessionTokenGuard'
-import { useRedeemedCampaignsQuery } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
 import { getUtmParamsFromCookie, TrackAction } from 'utils/tracking/tracking'
-import {
-  DownloadAppHedvigForeverMember,
-  DownloadAppRegular,
-} from './sections/DownloadApp'
-
-const TopBarWrapper = styled('div')`
-  color: ${colorsV3.black};
-`
+import { DownloadPageContent } from './components/DownloadPageContent'
 
 export const Download: React.FC = () => {
-  const campaigns = useRedeemedCampaignsQuery()
   const textKeys = useTextKeys()
 
-  if (campaigns.loading) {
```

Comment:
We don't have any referrals here?

Reply:
Ah, no the way I interpreted the old code is that we had some different copy if it was a referral, but as I wrote in the description the new design is just the same for everyone. So I removed it.

Reply:
sry missed that part in the description but okey the copy should be the same regardless so nothing important in DownloadAppHedvigForeverMember? (Did not check that file 😆 )

Reply:
OK I'll check again - I can see why you would want to really make sure I know what I'm doing when I remove stuff haha! 🙈

Reply:
No, `DownloadAppHedvigForeverMember` just had some other copy and for some reason a check which kind of mobile device the user's on so that the only link rendered would be the one to the right app place (like if you're on iPhone you would get one link - to App Store). Which I don't get why we had _only_  for referrals 🤷‍♀️  But feel free to check `src/client/pages/Download/sections/DownloadApp.tsx` out for yourself to see if I missed something.

Reply:
I thought maybe there was some referral special logic, but as you say  basically just copy and some conditional rendering which app icon to display. Haha its good to sanity check when removing stuff 😆